deploy/scripts/src/authentication/test.ts

```diff
@@ -265,118 +273,180 @@ export function signUp (): void {
 
   sleep(1)
 
-  res = group('B01_SignUp_11_ContinueAccountCreated POST', () =>
-    timeRequest(() => res.submitForm({
-      params: { tags: { name: 'B01_SignUp_11_ContinueAccountCreated' } }
-    }), { isStatusCode200, ...pageContentCheck('User information') }))
+  group('B01_SignUp_11_ContinueAccountCreated POST', () => {
+    res = timeRequest(() => res.submitForm({
+      params: {
+        redirects: 1,
+        tags: { name: 'B01_SignUp_11_ContinueAccountCreated_01_AuthCall' }
+      }
+    }),
+    { isStatusCode302 })
+    res = timeRequest(() => http.get(res.headers.Location, {
+      redirects: 0,
+      tags: { name: 'B01_SignUp_11_ContinueAccountCreated_02_OIDCCall' }
+    }),
+    { isStatusCode302 })
+    res = timeRequest(() => http.get(res.headers.Location, {
+      tags: { name: 'B01_SignUp_11_ContinueAccountCreated_03_RPStub' }
+    }),
+    { isStatusCode200, ...pageContentCheck('User information') })
+    iterationsCompleted.add(1)
+  })
 
   // 25% of users logout
   if (Math.random() <= 0.25) {
     sleep(1)
-
     res = group('B01_SignUp_12_Logout', () =>
       timeRequest(() => res.submitForm({
         params: { tags: { name: 'B01_SignUp_12_Logout' } }
       }), { isStatusCode200, ...pageContentCheck('Successfully signed out') }))
   }
-  iterationsCompleted.add(1)
```

Comment:
Keep this at the end of the function

Reply:
The `logout` step is purely a stub call so this metric has been moved before that step to provide an accurate view of the number of `sign-ups` and `sign-ins`.

Reply:
If that last step is a stub call just remove it from the script entirely. This metric increment should be the last step in the function regardless

Reply:
Slight correction there. The logout step has a redirect call to Orchestrator. I have updated this to split the stub and OIDC calls in the logout step.
Also, the iterationCompleted metric has been moved at the end of the function.